src/bash/initenv.sh

```diff
@@ -1,18 +1,19 @@
 #!/bin/bash
 
 ###
-# This is a script for fetching and installing the transifex client
+# This is a script for fetching and installing the jira client
 # within a virtualenv, and should be environment-dependancy free.
 # We download the virtualenv distribution, create an environment
 # and install the jira client within that environment
-#
-# The transifex client command can then be called from other tools.
 ###
 
 ### SETUP ###
 # virtualenv version
 VIRTUALENV_VERSION=15.0.1
 
+# other libs versions
+PYYAML_VERSION=3.12
```

Comment:
Since we are changing the environment we should set up, we should bump the `VERSION` specified in main.sh as well so that the env get's re-set up for anyone who has already created one.